contrib/test/integration/main.yml

```diff
@@ -118,3 +122,16 @@
           crio_socket: "/var/run/crio/crio.sock"
     - name: run k8s e2e tests
       include: e2e.yml
+
+# N/B: This is done after all testing, so it doesn't get in the way.
+- hosts: all
+  remote_user: root
+  vars_files:
+    - "{{ playbook_dir }}/vars.yml"
+  tags:
+    - always
```

Comment:
this requires a change in another repository to make it working, have you already created another patch on said repo?

Reply:
I don't remember if it's openshift/release or aos-cd-jobs, please check that and open the relevant PR over there so we can land this asap

Reply:
The `always` tag is magic.  It "always" runs no matter what tags are specified on command-line.  Last I checked this part was working as written, not sure about latest runs, will check soon.